src/js/server/api.js

```diff
@@ -104,7 +106,7 @@ fluid.defaults("fluid.express.user.api", {
         reset: {
             type: "fluid.express.user.reset",
             options: {
-                priority: "after:session"
+                priority:   "after:session"
```

Comment:
Is this whitespace change intentional?

Reply:
it was indented with its previous neighbours that I removed.  I'll update it.